v1/pipeline/common/harness-file-store-source.yaml

```diff
@@ -5,6 +5,9 @@ allOf:
   properties:
     file:
       type: string
+      metadata:
+        inputProperties:
+          dependsOn: [ /spec.source.type ]
```

Comment:
Hi @srdjan-harness currently the BE implementation uses the `/` as seperator instead of `.`.